astropy/utils/xml/check.py

```diff
@@ -55,9 +55,9 @@ def check_mime_content_type(content_type):
     (syntactically at least), as defined by RFC 2045.
     """
     ctrls = ''.join(chr(x) for x in range(0, 0x20))
-    token_regex = '[^()<>@,;:\\\"/[\]?= %s\x7f]+' % ctrls
+    token_regex = '[^()<>@,;:\\\"/[\]?= {}\x7f]+'.format(ctrls)
```

Comment:
Yes, they made me anxious too, but these were the simpler ones unlike some other modules. I had hundreds of failures at the beginning, so am really hoping that most of the issues got weeded out already.